features/Kiesrecht.feature

```diff
@@ -1,38 +1,101 @@
 	Als de ingeschreven persoon wordt geraadpleegd
 	Dan is europeesKiesrecht gelijk aan true
 
-Scenario: de ingeschreven persoon is niet uitgesloten van europees kiesrecht (2)
+Scenario: de ingeschreven persoon is niet uitgesloten van Europees kiesrecht
 	Gegeven op de PL van een ingeschreven persoon is categorie 13, element 31.10 is leeg
 	Als de ingeschreven persoon wordt geraadpleegd
+	Dan is europeesKiesrecht niet aanwezig
+
+Scenario: de ingeschreven persoon is uitgesloten van Europees kiesrecht en de einddatum uitsluiting Europees kiesrecht ligt in het verleden
+	Gegeven op de PL van een ingeschreven persoon is categorie 13, element 31.10 gelijk aan 1 (= persoon is uitgesloten)
+	En categorie 13, element 31.30 is een datum in het verleden (20180523)
 	Dan is europeesKiesrecht gelijk aan true
+	En is einddatumUitsluitingEuropeesKiesrecht niet aanwezig
 
-Abstract Scenario: de einddatum uitsluiting europees kiesrecht is geheel of deels onbekend
-	Gegeven op de PL van een ingeschreven persoon is categorie 13, element 31.30 gelijk aan <geheel of deels onbekende datum>
+Scenario: de ingeschreven persoon is uitgesloten van Europees kiesrecht en de einddatum uitsluiting Europees kiesrecht ligt in de toekomst
+	Gegeven op de PL van een ingeschreven persoon is categorie 13, element 31.10 gelijk aan 1 (= persoon is uitgesloten)
+	En categorie 13, element 31.30 is een datum in de toekomst (20500523)
+	Dan is europeesKiesrecht gelijk aan false
+	En is einddatumUitsluitingEuropeesKiesrecht.datum gelijk aan 2050-05-23
+	En is einddatumUitsluitingEuropeesKiesrecht.jaar gelijk aan 2050
+	En is einddatumUitsluitingEuropeesKiesrecht.maand gelijk aan 5
+	En is einddatumUitsluitingEuropeesKiesrecht.dag gelijk aan 23
+
+Scenario: de einddatum uitsluiting europees kiesrecht is geheel onbekend
+	Gegeven op de PL van een ingeschreven persoon is categorie 13, element 31.10 gelijk aan 1 (= persoon is uitgesloten)
+	En op de PL van een ingeschreven persoon is categorie 13, element 31.30 gelijk aan 00000000
 	Als de ingeschreven persoon wordt geraadpleegd
-	Dan is einddatumUitsluitingEuropeesKiesrecht leeg
+	Dan is einddatumUitsluitingEuropeesKiesrecht niet aanwezig
+	En is europeesKiesrecht gelijk aan false
 
-	Voorbeelden: 
-	| geheel of deels onbekende datum |
-	| 00000000                        |
-	| 20180000                        |
-	| 20181000                        |
+Abstract Scenario: de einddatum uitsluiting europees kiesrecht is deels onbekend en in het verleden
+	Gegeven op de PL van een ingeschreven persoon is categorie 13, element 31.10 gelijk aan 1 (= persoon is uitgesloten)
+	En op de PL van een ingeschreven persoon is categorie 13, element 31.30 gelijk aan <geheel of deels onbekende datum>
+	Als de ingeschreven persoon wordt geraadpleegd
+	Dan is einddatumUitsluitingEuropeesKiesrecht niet aanwezig
+	En is europeesKiesrecht niet aanwezig

```

Comment:
Is dit correct? Moet het niet gelijk zijn aan false?

Reply:
Waarom verwacht je hier false? Wat we weten is dat er een uitsluiting was, maar dat die niet meer geldt (einddatum uitsluiting ligt in het verleden).

Reply:
Ik lees <geheel of deels onbekende datum> als placeholder en ik had niet naar de voorbeelden gekeken waar alleen deels onbekende datum staan. In dit geval is dus de naam van de placeholder niet correct. De placeholder moet worden <deels onbekende datum>